alpine/Dockerfile

```diff
@@ -12,23 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM python:3.6-alpine3.8
+FROM python:3.6-alpine3.10
 
 ENV PYTHONUNBUFFERED=1
 
-RUN apk add --update --no-cache openssl ca-certificates
-
+# Work in the /opt directory to persist the pip-compiled requirement list for
+# child images to use.
 WORKDIR /opt
 
-COPY wsgi-requirements.in ./
+RUN apk update
+
+RUN apk add --no-cache \
+    # Update OpenSSL and certificates
+    openssl \
+    ca-certificates \
+    # Libraries required for psycopg2
+    postgresql-dev \
+    python3-dev \
```

Comment:
:+1: I'll remove it. Not a fan of circumventing the distro's packaging system though.

Reply:
I'm getting the same error as below when `python3-dev` is not installed:

    ImportError: Error loading shared library libpq.so.5: No such file or directory (needed by /usr/local/lib/python3.6/site-packages/psycopg2/_psycopg.cpython-36m-x86_64-linux-gnu.so)

Do you have any advice as for how to use the preinstalled headers? I really feel the best approach here is to just install and keep the packages.

Reply:
I'll give this a test. I guess we would need `libpq-dev` at compile time and `libpq` only at runtime. I guess the headers wouldn't be too big so it might be easiest to keep them around.

Reply:
Okay, I think I got it sorted out but will comment on the postgres-base PR.